backend/src/libs/plugins/authorization/authorization.plugin.ts

```diff
@@ -0,0 +1,46 @@
+import fp from 'fastify-plugin';
+
+import { ExceptionMessage } from '#libs/enums/enums.js';
+import { AuthError } from '#libs/exceptions/exceptions.js';
+import { isWhiteRoute } from '#libs/packages/server-application/libs/helpers/helpers.js';
```

Comment:
```suggestion
import { isWhiteRoute } from '#libs/packages/server-application/server-application.js';
```
Same for other places as well